src/pkg/projects/netcoreappRIDs.props

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <OfficialBuildRID Include="linux-x64" />
     <OfficialBuildRID Include="rhel.6-x64" />
+    <OfficialBuildRID Include="alpine.3.6-x64" />
```

Comment:
I think we will need CoreCLR and CoreFX packages for `alpine.3.6-x64` before we can check this in.

Reply:
Ah, I didn't know having that in would cause troubles, I'll remove it and put that in after we have the official ones. Is it the same case for corefx / coreclr? I've already merged this in into coreclr.

Reply:
I think you should be fine putting this in for corefx and coreclr.

The difference here is that core-setup needs to restore packages from coreclr and corefx for all those RIDs in order to build its "platform manifest" file.